packages/tools/client-debugger/client-debugger-view/src/test/widgets/EmojiButton.tsx

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { TooltipHost } from "@fluentui/react";
+import { Button, useId } from "@fluentui/react-components";
+import React from "react";
+
+import { SharedCell } from "@fluidframework/cell";
+
+/**
+ * {@link EmojiButton} input props.
+ */
+export interface EmojiButtonProps {
+	emojiCell: SharedCell<boolean>;
+}
+
+/**
+ * Simple button that displays either a smily or frouny emoji.
+ * Pressing the button toggles between the two.
+ *
+ * State is shared via the provided `SharedCell`.
+ */
+export function EmojiButton(props: EmojiButtonProps): React.ReactElement {
+	const { emojiCell } = props;
+
+	// undefined => No expression
+	// false => frouny
+	// true => smily
+	const [isSmily, setIsSmily] = React.useState<boolean | undefined>(emojiCell.get());
```

Comment:
Nits on the same vein, feel free to skip 😄 
```suggestion
	const [isSmiley, setIsSmiley] = React.useState<boolean | undefined>(emojiCell.get());
```